src/pages/AllocationPage/AllocationPage.tsx

```diff
@@ -247,6 +260,17 @@ export const AllocationPage = () => {
             </StepButton>
           </Step>
         ))}
+        <ApeInfoTooltip>
+          Reward your teammates in the circle by sending them GIVE tokens.
```

Comment:
instead of hardcoded 
```suggestion
          Reward your teammates in the circle by sending them {selectedCircle.tokenName} tokens.
```